src/dawn/Optimizer/ReorderStrategy.h

```diff
@@ -15,10 +15,11 @@
 #ifndef DAWN_OPTIMIZER_REORDERSTRATEGY_H
 #define DAWN_OPTIMIZER_REORDERSTRATEGY_H
 
+#include "dawn/Optimizer/OptimizerContext.h"
 #include <memory>
 
 namespace dawn {
-
+class OptimizerContext;
```

Comment:
I would not forward declare it if you include the file

Reply:
done